src/main/java/org/jitsi/jicofo/event/BridgeEvent.java

```diff
@@ -51,11 +51,36 @@
         = "org/jitsi/jicofo/JVB/UNHEALTHY";
 
     /**
+     * The event is emitted by
+     * {@link org.jitsi.impl.protocol.xmpp.colibri.ColibriConferenceImpl}
+     * when new video channels are being allocated which results in video stream
+     * count change. It is consumed by {@link BridgeSelector} to estimate
+     * the current video stream count for each JVB.
+     */
+    public static final String VIDEOSTREAMS_ADDED
+        = "org/jitsi/jicofo/JVB/VIDEOSTREAMS_ADDED";
+
+    /**
+     * The event is emitted by
+     * {@link org.jitsi.impl.protocol.xmpp.colibri.ColibriConferenceImpl}
+     * when video channels are being expired which results in video stream
+     * count change. It is consumed by {@link BridgeSelector} to estimate
+     * the current video stream count for each JVB.
+     */
+    public static final String VIDEOSTREAMS_REMOVED
+        = "org/jitsi/jicofo/JVB/VIDEOSTREAMS_REMOVED";
```

Comment:
Not a blocker, just a small suggestion to remove some switches and de-duplicate some code, so please feel to ignore: instead of having two events (VIDEOSTREAMS_REMOVED and VIDEOSTREAMS_ADDED) that carry a positive integer, we can have a single event VIDEOSTREAMS_CHANGED that carry a positive or a negative integer (the diff).